tw-tkms-starter/src/main/java/com/transferwise/kafka/tkms/config/TkmsProperties.java

```diff
@@ -238,6 +239,11 @@ public void afterPropertiesSet() {
    */
   private boolean deferMessageRegistrationUntilCommit = false;
 
+  /**
+   * Whether every message sent is required to have {@code x-wise-uuid} header. See {@link TkmsMessage#getUuid()} for more details.
+   */
+  private boolean uuidHeaderRequired = true;
```

Comment:
I understood that we want to start reminding people that they still need to do this after they set it to false.
Maybe an option is to make it into a date after which the library reminds the owners to look into this again. It probably needs to be limited as well to avoid setting this too much into the future.
We use a similar thing in WJP and so far it seems to work as intended.

Reply:
Thanks. I think it actually makes sense to start with less radical approach and don't require anything atm. Made a change to remove such requirement.